From 754569bf1448d515f507ec55bda731a7879cedb4 Mon Sep 17 00:00:00 2001
From: Ariel Aviad <aaviad@habana.ai>
Date: Thu, 13 Mar 2025 13:46:38 +0200
Subject: [PATCH] [SW-221257] g3: add littlefs image for serdes configuration

Add LittleFS file system image of guadi3 serdes
configuration.
This is preparing to load the image to the
target and store it in the flash

Change-Id: I8def9b40b6c9125c512b3e7069b19920c7b04eea
Signed-off-by: Ariel Aviad <aaviad@habana.ai>
---

diff --git a/board/habanalabs/gaudi3/serdes/FlashMemory.bin b/board/habanalabs/gaudi3/serdes/FlashMemory.bin
new file mode 100644
index 0000000..aaa9aa4
--- /dev/null
+++ b/board/habanalabs/gaudi3/serdes/FlashMemory.bin
Binary files differ
